environments/azure.yaml

```diff
@@ -83,7 +83,7 @@ isapi:
           - /identity/css/*  
           - /identity/nhsuk-frontend/*          
           - /identity/account/*          
-          - /identity/consent/*          
+          - /identity/consent*          
           - /identity/error/*
           - /identity/connect/*
           - /identity/.well-known/*
```

Comment:
Should we change the paths to match the pattern in all other instances?

Reply:
that was the one I *had* to change

Reply:
Are the other ones going ok through the ingress then?

Reply:
they seem to be. I wanted to minimise change at this point.

Reply:
👍 Just wanted to check